src/main/java/com/endava/internship/collections/StudentList.java

```diff
@@ -1,40 +1,53 @@
 package com.endava.internship.collections;
 
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.List;
-import java.util.ListIterator;
+import java.util.*;
```

Comment:
it is good for code readability to do not use wildcard imports

Reply:
Ok, I will change it